README.md

```diff
@@ -15,6 +15,9 @@ programming and debugging. These are the main features:
   to use 64-bit registers and opcodes with it)
 * Can be developed with newer-generation emulators (cen64, Ares, Dillonb's n64,
   m64p) or development cartridges (64drive, EverDrive64).
+* Support both vanilla N64 and iQue Player (chinese variant). The support is
```

Comment:
Saying that "**all** ROMs will work on both systems" sounds a bit overpromising, it should probably still be considered unfinished and experimental for now. Also, even when it's more mature it will probably still depend on which features are used and if the ROM does anything stupid, buggy or just something out of the ordinary API usage patterns.

Reply:
That's true but it's also overly generic. Can we say that all libdragon ROMs work on vanilla N64 either? Not at all, obviously the programmer can always do something that prevents them from working.

I guess it's more a styling issue here. The concept to convey is that you don't need to do anything specific to make ROMs work on iQue. Can you suggest a phrasing that avoids the word "all" which seems to be the one that can be misunderstood?

Reply:
How about "all current libdragon example ROMs and integration tests"

Reply:
That's missing the information that we had not to change the examples to make them work. That is, what I would want to say is that the design choice is that user code shouldn't change at all to run on ique

Reply:
I've sent a new wording, does this sound better?

Reply:
Sounds good enough to me.